### PR TITLE
fix: #420 signup debug

### PR DESCRIPTION
**Pull request by wjtwzgl12**
fix signup debug

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
   "dependencies": {
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
+    "@fortawesome/fontawesome-free": "^6.6.0",
+    "@mui/material": "^5.16.4",
     "async-validator": "^4.2.5",
```
Comment (ucfyao) [inline package.json:18]: what is mui used for?

File: src/app/signup/page.tsx [hunk 2]
```diff
@@ -259,4 +340,47 @@ const Signup = () => {
...
+  .card-footer {
+    background-color: #f7f7f7;
+    border-top: none;
+  }
+
+  .feature {
+    padding: 5rem;
+  }
+
+  .forget-password {
```
Comment (Yingxue0323) [inline src/app/signup/page.tsx:371]: This css is never used, why it's here?

File: package.json (hunk 1 above)
Comment (wjtwzgl12) [reply package.json:18]: for alert Components

File: src/app/signup/page.tsx (hunk 2 above)
Comment (wjtwzgl12) [reply src/app/signup/page.tsx]: removed

**Pull request merged**